src/WebKit_Performance_001.test.js

```diff
@@ -0,0 +1,88 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  webKitBrowserActions,
+  settingUrl,
+} from './commonMethods/commonFunctions'
+import {
+  validateFPS,
+  initializeSamplesArray,
+  fetchFPS,
+  calcAvgFPS,
+} from './commonMethods/webKitPerformanceCommonFunctions'
+import constants from './commonMethods/constants'
+
+const URL = 'http://webkit.org/blog-files/3d-transforms/poster-circle.html'
+let minFPS = 40
+
+export default {
+  setup() {
+    initializeSamplesArray()
+  },
+  title: 'WPEWebkit performance poster circle',
+  description: 'Loads the Poster Circle CSS3 animation and measures its performance',
+  steps: [
+    {
+      description: 'Deactivating monitor',
```

Comment:
I don't think we have to deactivate / active the monitor anymore. This was added at a time where the Monitor was giving issues during the test execution.

Reply:
Updated the code as per the comment